Code/Engine/Core/World/Implementation/WorldData.h

```diff
@@ -241,6 +241,7 @@ namespace ezInternal
     mutable ezAtomicInteger32 m_iReadCounter;
 
     ezUInt32 m_uiUpdateCounter = 0;
+    bool m_bDesiredSimulateWorld = true;
```

Comment:
Typically we would name a variable something with "should" or "want". I think "m_bShouldSimulateWorld" sounds good and easy to understand.